src/rule_definition_tools.jl

```diff
@@ -136,11 +136,11 @@ function rule_from_partials(∂s...)
                 push!(∂_mul_Δs_conjugate, ∂_mul_Δ)
             end
         end
-        primal_rule = :(Rule($Δs_tuple -> add($(∂_mul_Δs_primal...))))
+        primalgit remote add upstream_rule = :(Rule($Δs_tuple -> add($(∂_mul_Δs_primal...))))
```

Comment:
This seems ... wrong

Reply:
Yes, that was my editor. 😁 Should be fixed.